### PR TITLE
feat: remove deprecated input output values

### PR DESCRIPTION
**Pull request by sergebouchut2**
## Related issues

- https://github.com/Substra/substra-tests/pull/256
- https://github.com/Substra/substra/pull/361
- https://github.com/Substra/substrafl/pull/129

## Description

:warning: **DO NOT MERGE** until the frontend is switched to the new endpoints.

Following #496, removing deprecated field.
- input: addressable, permissions
- output: values

Fixes FL-568

## Checklist

- [x] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203097915891016

### CODE REVIEW

**Review by cboillet-dev — APPROVED** [review]

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Review by oleobal — APPROVED** [review]
Do we need to wait until 0.26 is out before merging?

File: backend/api/tests/views/test_views_computetask.py [hunk 1]
```diff
@@ -563,7 +556,8 @@ def test_task_match(self):
         # this should be enough to guarantee that there will only be one matching task
         params = urlencode({"match": key[19:]})
         response = self.client.get(f"{self.url}?{params}", **self.extra)
-        self.assertEqual(
+        print(self.list_expected_results[0].get("function"))
```
Comment (SdgJlbl) [inline backend/api/tests/views/test_views_computetask.py:559]: 🕵️

**Pull request merged**